include/z64camera.h

```diff
@@ -11,15 +11,14 @@
 #define CAM_STAT_UNK100     0x100
 
 #define NUM_CAMS 4
-#define MAIN_CAM 0
-#define SUBCAM_FIRST 1
-#define SUBCAM_FREE 0
-#define SUBCAM_NONE -1
-#define SUBCAM_ACTIVE -1
+#define CAM_ID_MAIN 0
+#define CAM_ID_SUB_FIRST 1
+#define CAM_ID_NONE -1
+#define CAM_ID_ACTIVE -1
```

Comment:
I didn't notice this in my first review, but I feel like these defines should really be the same thing. From what I can tell, `CAM_ID_ACTIVE` is always interpreted as "use active cam", but I think we could also see this as a default behavior when running into `CAM_ID_NONE`, and just use that define all the time.

Also, it would be okay to have defines sharing the same value if they were used strictly during different scenarios, but it's not really the case here. In fact, there are a lot of sequences like `Gameplay_CreateSubCamera`, which may return -1, followed directly by `Gameplay_ChangeCameraStatus`, which has specific behavior based on -1, so using different defines doesn't really fit imo.